TheJumpCloud/jumpcloud-ADMU
# Sa 3609 admu gha migrate ci ken

## Issues
* [SA-3609](https://jumpcloud.atlassian.net/browse/SA-3609) - ADMU CI Migration-Split Changes

## What does this solve?
Split tests that are not using JCAgent for quicker test times.

## Is there anything particularly tricky?
n/a
## How should this be tested?
Read through the changes and make sure tests are not missed or misplaced.


[SA-3609]: https://jumpcloud.atlassian.net/browse/SA-3609?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
